addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -12,6 +12,7 @@
     .o_dropdown_button,
     .o_datepicker_button {
         @include print-variable(o-input-border-color, $o-form-lightsecondary);
+        color: $o-main-text-color !important;
```

Comment:
Seems either unrelated, either not explained in the commit message ?

Reply:
it's related to the requirment check (it kept the red color)

I add a note about that in the commit message